2-WebApp-graph-user/2-5-HybridFlow/README.md

```diff
@@ -0,0 +1,771 @@
+
+  ```json
+  {
+    // ... 
+    "AzureAd": {
+      // ... 
+      "Certificate":  {
+        "SourceType":  "KeyVault",
+        "KeyVaultUrl":  "<YOUR_KEY_VAULT_URL>",
+        "KeyVaultCertificateName":  "<YOUR_KEY_VAULT_CERTIFICATE_NAME>"
+      }
+    }
+  }
+  ```
+  
+  3. If you had set `ClientSecret` previously, change its value to empty string, `""`.
+
+</details>
+
+### Step 4: Running the sample
+
+ For command line run the next commands:
+
+```console
+    cd 2-WebApp-graph-user\2-5-HybridFlow
+    dotnet run
+```
+
+## Using the sample
+
+<details>
+ <summary>Expand to see how to use the sample</summary>
+ Open your web browser and navigate to `https://localhost:7089`. You should see the application home page with a link for **Home**, **Privacy** and **Sign in**. Click the **Sign in** link and you'll be redirected to the Microsoft login page. Sign in using an user account in your tenant. After you sign in the client side `MSAL.js` client will receive an **auth** code from the server that will be exchanged for an authorization token and cached immediately in the browser.
+
+1. Click on `Profile` tab to see the signed in user's information and contacts. This information is retrieved using the `MSAL.js` library in the browser.
+
+Did the sample not work for you as expected? Did you encounter issues trying this sample? Then please reach out to us using the [GitHub Issues](../../../../issues) page.
+
+[Consider taking a moment to share your experience with us.](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbRz0h_jLR5HNJlvkZAewyoWxUNEFCQ0FSMFlPQTJURkJZMTRZWVJRNkdRMC4u)
+</details>
+
+## About the code
+
+<details>
+ <summary>Expand to see more information about the code</summary>
+
+## Server setup
+
+The entire application is built using the [Microsoft Identity Web](https://docs.microsoft.com/azure/active-directory/develop/microsoft-identity-web) library and [ASP.NET Core](https://docs.microsoft.com/aspnet/core/introduction-to-aspnet-core) using [Razor](https://docs.microsoft.com/aspnet/web-pages/overview/getting-started/introducing-razor-syntax-c) pages.
+
+In other samples you may have noticed that samples leveraged a class called `AuthenticationConfig` to bind the settings in the `appsettings.json` file into an `IConfiguration` object to make the settings available throughout the application.
+
+Using the [options pattern in ASP.Net Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-6.0) the configurations set in `appsettings.json` can instead be bound directly into `Options` objects and injected directly into the application.
```

Comment:
>?view=aspnetcore-6.0 [](http://example.com/codeflow?start=123&length=20)

Always drop versions from docs.microsoft.com links. That way they always redirect to the latest version